### PR TITLE
Updating dependencies to maintain compatibility with mac

### LINKED ISSUE #51: Error during npm install
```
npm ERR! Darwin 15.2.0
npm ERR! argv "/usr/local/Cellar/node/5.3.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.3.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE

npm ERR! bcrypt@0.8.5 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bcrypt@0.8.5 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the bcrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls bcrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/filo/drive/workspace/nodeapi/npm-debug.log
```

### PR DESCRIPTION
**Pull request by chrisgorgo**
Closes #51.

### CODE REVIEW

**Comment by cdaringe** [comment]
thanks Chris!  we'll merge it as soon as our production servers and services are all ready to run the latest node LTS

File: package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "grunt-exec": "0.4.6",
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
+    "grunt-cli": "0.1.13",
```
Comment (dylancwood) [inline package.json]: We typically install the `grunt-cli` globally: AFAIK, this is the community-standard-practice. Let me know if there's a reason that you can't do that.

Comment (chrisgorgo) [inline package.json]: I just wanted the install process to be slightly less painful. If there is no way to indicate in package.json that this package needs to be installed globally this prerequisite needs to be added to the README.

Comment (dylancwood) [inline package.json]: That sounds reasonable. We're coming from a place where all COINS developers' global NPM packages are managed for them by Ansible, so they have never been missing `grunt-cli`. Adding it to the README seems like the best idea.

**Comment by cdaringe** [comment]
ref: https://github.com/MRN-Code/ansible-ops/pull/19/files

**Comment by dylancwood** [comment]
@chrisfilo: 

Would you be willing to modify your PR to just include the version bump to mcrypt? If you do, I can merge it directly. Of course, we can just bump it ourselves, but I would like to merge your PR, since you went to the trouble of submitting it :+1:

File: package.json [hunk 2]
```diff
@@ -45,7 +45,8 @@
     "redis": "1.0.0",
     "relations": "0.5.0",
     "uuid": "2.0.1",
-    "vision": "3.0.0"
+    "vision": "3.0.0",
+    "boom": "2.8.0"
```
Comment (dylancwood) [inline package.json:49]: Boom is already present on [line 16](https://github.com/chrisfilo/nodeapi/blob/develop/package.json#L16)

**Comment by dylancwood** [comment]
@chrisfilo: I'm just going to merge as is :+1:

**Comment by chrisgorgo** [comment]
Thanks!

**Pull request merged**